src/components/Button.tsx

```diff
@@ -122,7 +122,7 @@ const colorClasses = ({
   )
 
 const textGradient = ({ type, loading, disabled }: ButtonProps) =>
-  type === 'tertiary'
+  type === 'tertiary' || type === 'secondary'
```

Comment:
This doesn't look right. If it's the same style, don't create an extra `type`

Reply:
Removed. Fixed